config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :categories do
+    resources :spends
+  end
```

Comment:
- [Optional] Dear @evillaz, since every category belongs to a user, I would suggest wrapping your category route with the  `resources :user`. This would make your routes more meaningful.